network/secure_session_wrapper.go

```diff
@@ -152,20 +153,28 @@ func (err *ConnectionWrapError) Timeout() bool {
 // Temporary always true
 func (err *ConnectionWrapError) Temporary() bool {
 	return true
-	//netErr, ok := err.error.(net.Error)
-	//if ok {
-	//	return netErr.Temporary()
-	//}
-	//return false
 }
 
+// SECURE_SESSION_ESTABLISHING_TIMEOUT timeout for secure session handshake that should be enough
+const SECURE_SESSION_ESTABLISHING_TIMEOUT = time.Second * 10
```

Comment:
no. first value was 2 sec but then decide to leave 10 sec for slow connections. It's rare case when it will need. only on network errors when it will not fail on earlier stages (wrap/unwrap, searching public key key)